.taskcluster.yml

```diff
@@ -98,15 +98,26 @@ tasks:
           each(browser):
             $map:
               - name: wpt-${browser.name}-${browser.channel}-stability
+                checkout: FETCH_HEAD
+                diff_range: HEAD^
                 description: >-
                   Verify that all tests affected by a pull request are stable
                   when executed in ${browser.name}.
                 extra_args: '--verify'
               - name: wpt-${browser.name}-${browser.channel}-results
+                checkout: FETCH_HEAD
+                diff_range: HEAD^
                 description: >-
                   Collect results for all tests affected by a pull request in
                   ${browser.name}.
                 extra_args: '--no-fail-on-unexpected --log-wptreport=../artifacts/wpt_report.json'
+              - name: wpt-${browser.name}-${browser.channel}-results-without-patch
```

Comment:
Maybe ...-master-results for the name?

Reply:
I don't think we can say the target branch is always master (one can open a PR against any branch, and `FETCH_HEAD^` will always be that target branch). Besides, "without patch" is the canonical suffix in Chromium CI for tasks like this (but I'm not sure whether non-Chromies would be able to immediately tell the meaning of it).

Reply:
I called it `sans_changes` and "without changes" in https://github.com/web-platform-tests/wpt/pull/14156, so `-without-changes` here would work, I can align with that.

Reply:
Alright, I'll go with "without changes".